rust/main/config/mainnet_config.json

```diff
@@ -3873,6 +3874,7 @@
       "testRecipient": "0xbB22547D1dc681fe925f568f637Ff67aC06c20fc",
       "validatorAnnounce": "0xb4fc9B5fD57499Ef6FfF3995728a55F7A618ef86",
       "index": {
+        "chunk": 30,
```

Comment:
this is so small 🥹 does it need to be this small?

Reply:
unfortunately so :(
```
(code: -32000, message: requested too many blocks from 39272316 to 39272422, maximum is set to 30, data: None)
```

Reply:
for reference we set it to 50 for hyperevm https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/7dbf7e4fa7aae231815efe03ea49470b7c805a49/rust/main/config/mainnet_config.json#L8594